instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/LambdaUtils.java

```diff
@@ -5,17 +5,25 @@
 
 package io.opentelemetry.instrumentation.awslambda.v1_0;
 
+import io.opentelemetry.api.metrics.GlobalMeterProvider;
+import io.opentelemetry.api.metrics.MeterProvider;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
 import io.opentelemetry.sdk.common.CompletableResultCode;
-import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
 import java.util.Arrays;
 import java.util.concurrent.TimeUnit;
 
 final class LambdaUtils {
 
   static void forceFlush(OpenTelemetrySdk openTelemetrySdk, long flushTimeout, TimeUnit unit) {
     CompletableResultCode traceFlush = openTelemetrySdk.getSdkTracerProvider().forceFlush();
-    CompletableResultCode metricsFlush = IntervalMetricReader.forceFlushGlobal();
+    MeterProvider meterProvider = GlobalMeterProvider.get();
```

Comment:
Is it expected that we pass here an instance of `OpenTelemetrySdk` but `MeterProvider` we take from the global? Why is it not possible to get the latter from the former?

Reply:
Until metrics is stable, we haven't associated it with the `opentelemetry-sdk` API at all, which is stable. So we have to access metrics separately - the model for autoconfigure is it sets up the SDK object but only creates a global meter provider, as a "side channel" into the alpha API. Eventually, it'll be available through the SDK object though